run_test.sh

```diff
@@ -31,4 +31,64 @@ function run_lint_fix {
   deactivate
 }
 
-run_lint_fix
\ No newline at end of file
+# Fixes python tests
+function run_py_test {
+  python3 -m venv .env
+  source .env/bin/activate
+  
+  cd simple
+  pip3 install -r requirements.txt
+
+  echo -e "#### Running stats tests"
+  python3 -m pytest tests/stats/ -s
+
+  deactivate
+}
+
+function run_all_tests {
+  run_lint_fix
```

Comment:
I realized the run_list_fix should not be run in cloudbuild.yaml and used in here. This option is meant to be run locally to fix the lint issues.

We need another `run_list_test` option to replace run_list_fix in cloudbuild.yaml and here that executes commands like https://github.com/datacommonsorg/website/blob/master/run_test.sh#L108-L122

Reply:
Done.